Dockerfile.supernode

```diff
@@ -13,10 +13,11 @@ FROM nginx:1.16-alpine
 
 RUN apk --no-cache add ca-certificates bash
 
+COPY --from=builder /go/src/github.com/dragonflyoss/Dragonfly/hack/start-supernode.sh /root/start.sh
 COPY --from=builder /go/src/github.com/dragonflyoss/Dragonfly/hack/supernode-nginx.conf /etc/nginx/nginx.conf
 COPY --from=builder /opt/dragonfly/df-supernode/supernode /opt/dragonfly/df-supernode/supernode
```

Comment:
Could we merge these commands into one command to reduce the number of image layers?

Reply:
Maybe it's not a good idea.  

+ Firstly, we copy the files from different directory to different directory, and it's difficult to merge these commands into one command.
+ Secondly, we can take advantage of cache of the static layer.